packages/ckeditor5-build-balloon-block/src/ckeditor.ts

```diff
@@ -32,82 +32,80 @@ import { CloudServices } from '@ckeditor/ckeditor5-cloud-services';
  */
 import '../theme/theme.css';
 
-export default class BalloonEditor extends BalloonEditorBase {}
+export default class BalloonEditor extends BalloonEditorBase {
+	public static override builtinPlugins = [
```

Comment:
Does the trick work because `builtinPlugins` have a tuple type with all plugin types?

Reply:
Yes, tuple uses types imported from plugin packages and this gives a TypeScript a chance to load module augmentations with them.